trace/exporter_type.go

```diff
@@ -33,6 +33,16 @@ func (t ExporterType) MarshalJSON() ([]byte, error) {
 	return []byte(`"` + t.String() + `"`), nil
 }
 
+func (t *ExporterType) UnmarshalJSON(b []byte) error {
```

Comment:
By convention, `"null" == string(b)` should be a noop.

Reply:
Done